csharp/ql/test/library-tests/cil/functionPointers/functionPointers.ql

```diff
@@ -0,0 +1,33 @@
+import cil
+import semmle.code.cil.Type
+
+bindingset[kind]
+private string getKind(int kind) { if kind = 1 then result = "modreq" else result = "modopt" }
+
+private string getAnnotatedType(Type t, Element e) {
```

Comment:
This should have `bindingset[t, e]` as it otherwise creates a Cartesian product.